draft-ietf-rats-corim.md

```diff
@@ -1217,14 +1217,18 @@ matched.
 ~~~
 
 The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
-Each `conditional-series-record` record is evaluated in the order it appears in the series array.
+
+Each `conditional-series-record` has measurements belonging to one or more measured elements, pertaining to the environment that appears in the stateful-environment record.
+
 The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the ACS.
```

Comment:
```suggestion

```

Reply:
Why have you deleted this?

Reply:
ok, understood the concept that in series, the first condition match terminates the series, to line 1224 is not really applicable!